features/fields.feature

```diff
@@ -341,3 +216,113 @@ Functionaliteit: Aanpasbare representatie met de fields parameter
        }
       }
     ```
+
+
+    Voor sommige API's kan besloten worden extra functionaliteit te bieden bij het invullen van de fields paramter. Bij de API specificaties en/of features is dan aangegeven dat de uitgebreide variant van de fields functionaliteit wordt geborden.
+    Deze functionaliteit maakt het mogelijk velden in groepen te vragen door alleen de naam van de property op te geven, of een deel van het pad, niet het hele pad ervoor. Dit kan nuttig zijn om de lengte van de waarde in fields te beperken.
+
+    Met de fields parameter kan een gevraagd veld worden aangeduid met:
+    - het hele pad naar dat veld, bijvoorbeeld fields=verblijfplaats.datumAanvangAdreshouding.jaar
+    - het laatste deel van het pad naar dat veld, bijvoorbeeld fields=datumAanvangAdreshouding.jaar
+    - alleen de veldnaam, bijvoorbeeld fields=jaar
+
+    Voor elk veld in fields moet dit leiden tot één uniek veld in de resource, ongeacht of dit veld bij de specifiek opgevraagde resource een waarde heeft. Bijvoorbeeld een persoon heeft geboorte.land en overlijden.land, dus is fields=land niet toegestaan. In dat geval wordt een foutmelding teruggegeven. In de foutmelding worden de gevonden velden teruggegeven in invalidParams.detail. Wanneer veel velden zijn gevonden (bijvoorbeeld meer dan 3), dan worden de eerste resultaten (bijv. de eerste 3 gevonden velden) gemeld.
+    Wanneer een veld in fields exact zo voorkomt in de resource, maar ook zo voorkomt als einde van het pad naar een ander veld, dan wordt het veld teruggegeven die exact wordt aangeduid met wat in fields is opgenomen.
+
+    Achtergrond:
+      Gegeven de resource zakelijkgerechtigden kent de volgende velden:
+      """
+      {
+        "identificatie",
+        "aanvangsdatum",
+        "erfpachtCanon": {
+          "soortErfpachtCanon": {
+            "code",
+            "waarde"
+          },
+          "jaarlijksBedrag": {
+            "som",
+            "valuta": {
+              "code",
+              "waarde"
+            }
+          }
+        },
+        "tenaamstelling": {
+          "aandeel": {
+            "noemer",
+            "teller"
+          },
+          "aantekeningen": [
+            {
+              "identificatie": "string"
+            }
+          ],
+          "gezamenlijkAandeel": {
+            "noemer",
+            "teller"
+          }
+        },
+        "persoon": {
+          "identificatie",
+          "omschrijving",
+          "type"
+        },
+        "_links": {
+          "self",
+          "persoon",
+          "betrokkenPartner"
+        }
+      }
+      """
+
+    Scenario: opvragen veld met fields door opgeven exacte veld in resource
+      Als zakelijkgerechtigden wordt gevraagd met fields=aanvangsdatum
```

Comment:
Moet fields niet expand zijn? In de Dan stap wordt _embedded genoemd. Maar volgens mij is dan de voorbeeld ook niet correct, ik moet toch bij expand aangeven welke subresource ik wil expanden?

Reply:
het gaat hier om een collectie van zakelijkgerechtigden

Reply:
OK, duidelijk. Ik redeneerde vanuit endpoint /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie} en niet vanuit /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}/zakelijkgerechtigden